src/store/modules/order/actions.ts

```diff
@@ -33,6 +33,28 @@ const actions: ActionTree<OrderState, RootState> = {
     }
     return resp;
   },
+  async getOrderDetail({ commit, state }, { payload, orderId }) {
+    let resp;
+    const current = state.current as any
+    if (current.length && current[0]?.doclist && current[0]?.doclist.docs[0].orderId === orderId)
```

Comment:
Improve the schema, current should have orderId and other common attribute while items should have complete list with total